soh/soh/Enhancements/debugger/colViewer.cpp

```diff
@@ -57,42 +57,75 @@ void ColViewerWindow::DrawElement() {
+
+    if (UIWidgets::EnhancementCheckbox("Collision Goggles", "gCollisionGoggles")) {
+        if (CVarGetInteger("gCollisionGoggles", 0)) {
+            CVarSetInteger("gColViewerEnabled", 1);
+            CVarSetInteger("gColViewerDecal", 0);
+            CVarSetInteger("gColViewerShaded", 1);
+            CVarSetInteger("gColViewerScene", COLVIEW_SOLID);
+            CVarSetInteger("gColViewerBgActors", COLVIEW_SOLID);
+            CVarSetInteger("gColViewerColCheck", COLVIEW_SOLID);
+            CVarSetInteger("gColViewerWaterbox", COLVIEW_TRANSPARENT);
+            CVarSetInteger("gColViewerColorACR", 5);
+            CVarSetInteger("gColViewerColorACG", 130);
+            CVarSetInteger("gColViewerColorACB", 100);
+        } else {
+            CVarSetInteger("gColViewerEnabled", 0);
+            CVarSetInteger("gColViewerDecal", 1);
+        }
+    }
+    UIWidgets::Tooltip(
+        "Disables rendering of the scene and actors, and instead only shows the collision of everything.");
+
+    if (!CVarGetInteger("gCollisionGoggles", 0)) {
+        UIWidgets::EnhancementCheckbox("Enabled", "gColViewerEnabled");
+
+        UIWidgets::LabeledRightAlignedEnhancementCombobox("Scene", "gColViewerScene", ColRenderSettingNames,
+                                                          COLVIEW_DISABLED);
+        UIWidgets::LabeledRightAlignedEnhancementCombobox("Bg Actors", "gColViewerBgActors", ColRenderSettingNames,
+                                                          COLVIEW_DISABLED);
+        UIWidgets::LabeledRightAlignedEnhancementCombobox("Col Check", "gColViewerColCheck", ColRenderSettingNames,
+                                                          COLVIEW_DISABLED);
+        UIWidgets::LabeledRightAlignedEnhancementCombobox("Waterbox", "gColViewerWaterbox", ColRenderSettingNames,
+                                                          COLVIEW_DISABLED);
+
+        UIWidgets::EnhancementCheckbox("Apply as decal", "gColViewerDecal", false,
+                                       "", UIWidgets::CheckboxGraphics::Cross, true);
+        UIWidgets::InsertHelpHoverText(
+            "Applies the collision as a decal display. This can be useful if there is z-fighting occuring "
+            "with the scene geometry, but can cause other artifacts.");
+        UIWidgets::EnhancementCheckbox("Shaded", "gColViewerShaded", false, "", UIWidgets::CheckboxGraphics::Cross,
+                                       true);
+        UIWidgets::InsertHelpHoverText("Applies the scene's shading to the collision display.");
+
+        // This has to be duplicated in both code paths due to the nature of ImGui::IsItemHovered()
+        const std::string colorHelpText = "View and change the colors used for collision display.";
+        if (ImGui::TreeNode("Colors")) {
+            UIWidgets::InsertHelpHoverText(colorHelpText);
+
+            UIWidgets::EnhancementColor("Normal", "gColViewerColorNormal", scene_col, ImVec4(255, 255, 255, 255),
```

Comment:
Not that you added these but i noticed they include a special 'lock' feature which prevents the use of randomise all from affecting it, but there is no such randomise all button. Since only here and the check tracker appear to use it maybe we can remove that since neither have a randomise all feature